docs/static/persistent-queues.asciidoc

```diff
@@ -1,96 +1,98 @@
 [[persistent-queues]]
 === Persistent Queues
 
-WARNING: This functionality is in beta and is subject to change. It should be deployed in production at your own risk.
+WARNING: This functionality is in beta and is subject to change. Deployment in production is at your own risk.
 
 By default, Logstash uses in-memory bounded queues between pipeline stages
-(input → filter and filter → output) to buffer events. The size of these 
-in-memory queues is fixed and not configurable. If Logstash terminates unsafely,
-either as the result of a software failure or the user forcing an unsafe
-shutdown, it's possible to lose queued events. 
+(inputs → pipeline workers) to buffer events. The size of these in-memory
+queues is fixed and not configurable. If Logstash terminates abnormally, the
+contents of the in-memory queue will be lost. Such abnormal terminations
+include software crash, `kill -9`, or the host machine loses power.
 
-To prevent event loss in these scenarios, you can configure Logstash to use
-persistent queues. With persistent queues enabled, Logstash persists buffered
-events to disk instead of storing them in memory. 
+Temporary machine failures are scenarios where Logstash or its host machine are
```

Comment:
I agree.

I've reorganized these two paragraphs to try and have a better flow to it.